papers/d2372r0.bs

```diff
@@ -63,6 +63,59 @@ and providing the `L` specifier to opt into localized formatting in the same way
 as it is done for all other standard formatters
 ([format.string.std](https://eel.is/c++draft/format.string.std)).
 
+Non locale-dependant specifiers
```

Comment:
I suggest renaming this section to `Locale alternative forms` because it talks about `O` modifier and since `%S` is actually locale-dependent.